app/utils/deep_link/index.ts

```diff
@@ -118,7 +118,12 @@ export function parseDeepLink(deepLinkUrl: string): DeepLinkWithData {
     try {
         const url = removeProtocol(decodeURIComponent(deepLinkUrl));
 
-        if (url.includes('../') || url.includes('/..')) {
+        if (
+            url.includes('../') ||
+            url.includes('/..') ||
+            url.includes('..%2f') ||
+            url.includes('%2f..')
```

Comment:
What about `%2f%2e.`, `%2f.%2e`, `/%2e.` and so forth?

Reply:
Added a second decoder and simplified